MBBSEmu/HostProcess/MbbsHost.cs

```diff
@@ -91,13 +93,18 @@ public class MbbsHost : IMbbsHost
         /// </summary>
         private readonly Timer _timer;
 
+        // Timer to prevent main loop from executing while cleanup is running
+        private readonly Timer _nightlyCleanupTimer;
+
+        private bool _performingNightlyCleanup;
```

Comment:
Add a comment/XML comment on the context for this variable as it's similar to others that are declared

Reply:
Complete.